test/test_datasets.py

```diff
@@ -210,6 +211,32 @@ def test_svhn(self, mock_check):
             dataset = torchvision.datasets.SVHN(root, split="extra")
             self.generic_classification_dataset_test(dataset, num_images=2)
 
+    @mock.patch('torchvision.datasets.voc.download_extract')
+    def test_voc_parse_xml(self, mock_download_extract):
+        with voc_root() as root:
+            dataset = torchvision.datasets.VOCDetection(root)
+
+            single_object_xml = """<annotation>
+              <object>
+                <name>cat</name>
+              </object>
+            </annotation>"""
+            multiple_object_xml = """<annotation>
+              <object>
+                <name>cat</name>
+              </object>
+              <object>
+                <name>dog</name>
+              </object>
+            </annotation>"""
+            single_object_parsed = dataset.parse_voc_xml(ET.fromstring(single_object_xml
+                ))
```

Comment:
This is a weird indenting (and I'm surprised that the linter doesn't complain), but well, doesn't block merging this PR, but could you fix it in a follow-up PR?